### PR TITLE
update: show route and bookstore schema

### LINKED ISSUE #97: Add social to bookstore object

### PR DESCRIPTION
**Pull request by VernL**
- Add show route
- Add social and address fields to Bookstore Schema

closes #97

### CODE REVIEW

File: backend/models/bookstore.js [hunk 1]
```diff
@@ -12,7 +12,16 @@ const BookstoreSchema = new mongoose.Schema({
   coffee: {type: Boolean, default: false},
   wifi: {type: Boolean, default: false},
   plugs: {type: Boolean, default: false},
-  events: {type: Boolean, default: false}
+  events: {type: Boolean, default: false},
+  twitter: String,
+  facebook: String,
+  instagram: String,
+  addressCountry: String,
```
Comment (RichardLitt) [inline backend/models/bookstore.js]: should be an object called address, with fields inside of it, no?

Comment (RichardLitt) [reply backend/models/bookstore.js]: Look for an express address schema module on npm. There ought to be one we can use.

Comment (VernL) [reply backend/models/bookstore.js]: I did not not find a schema module with a global reach. I updated the schema per schema.org.

File: backend/models/bookstore.js [hunk 2]
```diff
@@ -2,17 +2,33 @@ const mongoose = require('mongoose')
...
-  events: {type: Boolean, default: false}
+  Bookstore: {
+    name: String,
+    summary: String,
+    review: String,
+    thumbnail: String,
+    images: [String],
+    url: String,
+    telephone: String,
+    Amenities: {
```
Comment (RichardLitt) [inline backend/models/bookstore.js]: Convention is to not use capitals in JSON keys.

**Pull request merged**